src/settings/AboutDialog.ts

```diff
@@ -29,12 +33,21 @@ export class AboutDialog implements Component {
 				m("p", "© 2023 Tutao GmbH"),
 			]),
 			this._sendLogsLink(),
+			// wrap it in a div so that it's not filling the whole width
+			m(
+				"",
+				m(Button, {
+					label: () => "Show welcome dialog",
+					type: ButtonType.Primary,
+					click: vnode.attrs.onShowSetupWizard,
+				}),
+			),
```

Comment:
```suggestion
			app.isMobile() ? m(
				"",
				m(Button, {
					label: () => "Show welcome dialog",
					type: ButtonType.Primary,
					click: vnode.attrs.onShowSetupWizard,
				}),
			) : null,
```

This button is displayed also on platforms where the wizard is not used like web for instance.

Reply:
I have pushed this on mup's orders.